cargo-afl/src/main.rs

```diff
@@ -346,9 +346,8 @@ where
     let mut rustdocflags = rustflags.clone();
 
     rustflags.push_str(&format!(
-        "-l afl-llvm-rt \
-         -L {} ",
-        common::afl_llvm_dir().unwrap().display()
+        "-Clink-arg={} ",
+        common::object_file_path().unwrap().display()
```

Comment:
@vanhauser-thc This is the change I am least certain about. Perhaps you could give this a glance.

I'm going to run some more tests, but the fact that CI is passing is a good sign.

Reply:
this is how you would do it with an object file instead of an archive. LGTM